hipcheck/src/metric/mod.rs

```diff
@@ -11,18 +11,18 @@ pub mod fuzz;
 pub mod identity;
 pub mod linguist;
 mod math;
-pub mod review;
+
```

Comment:
My initial comment said "A lot of the lines..." and it seems you've only changed the two that were directly pointed at by the comment. Please go through the entire commit and remove random newlines that were added.